apm: add python tab to connect_dbm_and_apm docs (#16086)
DataDog/documentation: content/en/database_monitoring/guide/connect_dbm_and_apm.md

: [dd-trace-go][3] >= 1.44.0 (support for [database/sql][4] and [sqlx][5] packages)<br />
[dd-trace-rb][6] >= 1.6.0 (support for [mysql2][7] and [pg][8] gems)<br />

> [dd-trace-js][9] >= 3.9.0 or >= 2.22.0 (support for [postgres client][10])

Reviewer: ```suggestion
[dd-trace-js][9] >= 3.9.0 or >= 2.22.0 (support for [postgres client][10])<br />
```

{{% /tab %}}

{{% tab "Python" %}}

Update your app dependencies to include [dd-trace-py>=1.7.0][1]:
```
pip install "ddtrace>=1.7.0"
```

Install [psycopg2][2] (Note - DBM-APM linking is not currently supported for mysql clients):
```
pip install psycopg2
```

Enable the database monitoring propagation feature by setting the following enviornment variable:

>    `DD_TRACE_SQL_COMMENT_INJECTION_MODE=full`

Author: ```suggestion
   - `DD_TRACE_SQL_COMMENT_INJECTION_MODE=full`
```

For the best user experience ensure the following enviornemnts variables are set in your application:
   - `DD_SERVICE=(application name)`

>    - `DD_ENV=(application enviornment)`

Reviewer 2: ```suggestion
   - `DD_ENV=(application environment)`
```

>    `DD_SERVICE=(application name)`
>    `DD_ENV=(application enviornment)`
>    `DD_VERSION=(application version)`

Author: ```suggestion
   - `DD_SERVICE=(application name)`
   - `DD_ENV=(application enviornment)`
   - `DD_VERSION=(application version)`
```

```
pip install "ddtrace>=1.7.0"
```

Install [psycopg2][2] (Note - DBM-APM linking is not currently supported for mysql clients):
```
pip install psycopg2
```

Enable the database monitoring propagation feature by setting the following environment variable:
   - `DD_TRACE_SQL_COMMENT_INJECTION_MODE=full`

For the best user experience ensure the following environment variables are set in your application:
   - `DD_SERVICE=(application name)`
   - `DD_ENV=(application environment)`
   - `DD_VERSION=(application version)`

Full example:
```python

import psycopg2

#TODO: update postgres configurations
POSTGRES_CONFIG = {
    "host": "127.0.0.1",
    "port": 5432,
    "user": "postgres_user",
    "password": "postgres_password",
    "dbname": "postgres_db_name",
}

# connect to postgres db
conn = psycopg2.connect(**POSTGRES_CONFIG)
cursor = conn.cursor()
# execute sql queries
cursor.execute("select 'blah'")
cursor.executemany("select %s", (("foo",), ("bar",)))
```

[1]: https://ddtrace.readthedocs.io/en/stable/release_notes.html
[2]: https://ddtrace.readthedocs.io/en/stable/integrations.html#module-ddtrace.contrib.psycopg

> 

Author: ```suggestion

{{% /tab %}}

```

{{% /tab %}}

{{% tab "Python" %}}

Update your app dependencies to include [dd-trace-py>=1.7.0][1]:
```
pip install "ddtrace>=1.7.0"
```

> Install [psycopg2][2] (Note - DBM-APM linking is not currently supported for mysql clients):

Reviewer: ```suggestion
Install [psycopg2][2] (**Note**: Connecting DBM and APM is not supported for MySQL clients):
```

```
pip install psycopg2
```

Enable the database monitoring propagation feature by setting the following environment variable:
   - `DD_TRACE_SQL_COMMENT_INJECTION_MODE=full`

For the best user experience ensure the following environment variables are set in your application:
   - `DD_SERVICE=(application name)`
   - `DD_ENV=(application environment)`
   - `DD_VERSION=(application version)`

Full example:
```python

import psycopg2

> #TODO: update postgres configurations

Reviewer 2: Is the TODO meant for the end-user copying that example?

Author: Yes

[6]: https://github.com/dataDog/dd-trace-rb
[7]: https://github.com/brianmario/mysql2
[8]: https://github.com/ged/ruby-pg

> [9]: https://github.com/dataDog/dd-trace-py
> [10]: https://www.psycopg.org/docs/index.html
> [11]: https://github.com/DataDog/dd-trace-js
> [12]: https://node-postgres.com/

Author: ```suggestion
[9]: https://github.com/DataDog/dd-trace-js
[10]: https://node-postgres.com/
[11]: https://github.com/dataDog/dd-trace-py
[12]: https://www.psycopg.org/docs/index.html
```

[8]: https://github.com/ged/ruby-pg
[9]: https://github.com/DataDog/dd-trace-js
[10]: https://node-postgres.com/

> [11]: https://github.com/dataDog/dd-trace-py

Reviewer 2: ```suggestion
[11]: https://github.com/DataDog/dd-trace-py
```
